docs/features/joystick.md

```diff
@@ -18,6 +18,8 @@ By default the joystick driver is `analog`, but you can change this with:
 JOYSTICK_DRIVER = digital
 ```
 
+When using `analog` with ARM, [you must use 3.3v with your Joystick](../drivers/adc). Although ARM boards such as the [Helios](https://keeb.supply/products/0xcb-helios) have 5v pin output, the ADC driver does not support it.
```

Comment:
If this is noted on the adc driver page, I'd rather include a link with "important notes about ADC", or some such wording.

Reply:
I think a specific mention is important because

1. The ADC driver doc mentions that ARM analog pins only work on 3.3v, but with the joystick, it 'almost' works; it's behavior isn't something that indicates an issue with the driver or the wrong pin. If it didn't work completely, that's one thing, but 3/4 directions responded to input, and one axis was not functioning correctly. This lead me to thinking the joystick itself was defective or my board, not the ADC driver, especially because GP28 was reacting to input from GP29 in my case so I initially assumed that the board had a bad connection
2. If I saw a general note pointing me towards the ADC Driver doc, I would likely ignore it out of ignorance, my lack of EE knowledge leading me to think there's no way that could cause an issue. My initial logic was:
 * I've wired a joystick on a Elite-Pi (which is 3.3v only), so I know how it works in general
 * the joystick specs are 5v
 * my board has a 5v pin
 * I know the board defaults to analog
 * I know that the joystick should be on analog
 * I know the basic difference between analog and digital
 * I'm not trying to do anything out of the ordinary
 
    so in my mind there wouldn't be a reason to visit that page unless something like "the joystick may not work" was called out
4. ARM boards are becoming more popular, and typically if someone is going through the effort of adding something like a joystick they may be using 5v already to power other things such as LEDs, or the board may just have 5v output already like the helios